cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -354,37 +356,46 @@ protected void onCreate(Bundle savedInstanceState) {
   }
 
   /**
-   * Initializes the Java Bridge to allow communication between Java and JavaScript.
-   * This method injects Java objects into the WebView and loads corresponding JavaScript code.
+   * Initializes the Java Bridge to allow communication between Java and JavaScript. This method
+   * injects Java objects into the WebView and loads corresponding JavaScript code.
    */
   private void initializeJavaBridge() {
     Log.i(TAG, "initializeJavaBridge");
 
     WebContents webContents = getActiveWebContents();
     if (webContents == null) {
-        // WebContents not initialized yet, post a delayed runnable to check again
-        new Handler(Looper.getMainLooper()).postDelayed(new Runnable() {
-            @Override
-            public void run() {
-                initializeJavaBridge(); // Recursive call to check again
-            }
-        }, JAVA_BRIDGE_INITIALIZATION_DELAY_MILLI_SECONDS);
-        return;
+      // WebContents not initialized yet, post a delayed runnable to check again
+      new Handler(Looper.getMainLooper())
+          .postDelayed(
+              new Runnable() {
+                @Override
+                public void run() {
+                  initializeJavaBridge(); // Recursive call to check again
+                }
+              },
+              JAVA_BRIDGE_INITIALIZATION_DELAY_MILLI_SECONDS);
+      return;
     }
 
     // --- Initialize the Java Bridge ---
 
     // 1. Gather all Java objects that need to be exposed to JavaScript.
     javaScriptAndroidObjectList.add(new AmatiDeviceInspector(this));
+    javaScriptAndroidObjectList.add(new HTMLMediaElementExtension(this));
```

Comment:
If we are going to have more Java classes like this, should we consider to make a single Java class that stores all Java objects (and put comments to mention the purpose of the Java object) as a list, and result them here, so we can make the code cleaner?

Reply:
I think this should be a temporary solution to add javascript interface.

Reply:
Created b/379701165 and added a TODO.